### PR TITLE
 Fixing the build after travis upgraded Ubuntu

### PR DESCRIPTION
**Pull request by mistercrunch**
It seems Travis has been updated/ing its build environment to Ubuntu 16.04 from Ubuntu 14 in the past few days.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -66,22 +66,22 @@ jobs:
...
+        - postgresql
         - redis-server
       before_script:
         - psql -U postgres -c "CREATE DATABASE superset;"
         - psql -U postgres -c "CREATE USER postgresuser WITH PASSWORD 'pguserpassword';"
     - language: python
       python: 3.6
       env: TOXENV=pylint
-    - language: python
+    - language: java
```
Comment (etr2460) [inline .travis.yml]: How was this even working before?

File: .travis.yml [hunk 2]
```diff
@@ -66,22 +66,22 @@ jobs:
...
       python: 3.6
```
Comment (etr2460) [inline .travis.yml]: And is this needed anymore?

**Review by etr2460 — APPROVED** [review]
a couple questions, but lgtm otherwise

File: .travis.yml (hunk 1 above)
Comment (mistercrunch) [reply .travis.yml]: `jdk_switcher` just happened to be on the python image

File: .travis.yml [hunk 3]
```diff
@@ -16,6 +16,16 @@
...
+    - language: java
+      python: 3.6
+      env:
+        - TOXENV=license-check
+        - TRAVIS_CACHE=$HOME/.travis_cache/
+      jdk:
+        - openjdk8
+      install:
+        - sudo pip install --upgrade pip
+        - sudo pip install codecov tox
```
Comment (john-bodley) [inline .travis.yml]: I don't think `codecov` is needed here.

**Comment by john-bodley** [comment]
@mistercrunch it seems like the CI is failing because the license check is not running against Python 3.6 (I speculate it's Python 2.7) as it can't find the `black` package.

File: .travis.yml (hunk 3 above)
Comment (mistercrunch) [reply .travis.yml]: Sorry I had missed that and merged from mobile after @dpgaspar stacked commits that fixed the build on my PR...

**Pull request merged**